CMakeLists.txt

```diff
@@ -53,8 +64,6 @@ options are: None(CMAKE_CXX_FLAGS or CMAKE_C_FLAGS used) Debug Release
 RelWithDebInfo MinSizeRel.")
 endif()
```

Comment:
Kindly change to uniform indentation. Improves code readability

Reply:
done